tests/service/privacy_request/request_runner_service_test.py

```diff
@@ -207,22 +207,9 @@ def test_create_and_process_access_request_mssql(
 
     visit_key = result_key_prefix + "visit"
     assert results[visit_key][0]["email"] == customer_email
-    log_id = pr.execution_logs[0].id
-    pr_id = pr.id
     # Both pre-execution webhooks and both post-execution webhooks were called
     assert trigger_webhook_mock.call_count == 4
 
-    for webhook in policy_pre_execution_webhooks:
-        webhook.delete(db=db)
-
-    for webhook in policy_post_execution_webhooks:
-        webhook.delete(db=db)
-
-    policy.delete(db=db)
-    pr.delete(db=db)
```

Comment:
It's generally good practise to tidy up what's created as part of the test itself, which in this case `pr` is

Reply:
Fixed! Back over to you @seanpreston